sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -101,6 +185,10 @@ def test_feature_importances():
     mask = np.abs(transformer.estimator_.coef_) > 1e-5
     assert_array_equal(X_new, X[:, mask])
```

Comment:
This test is failing because of [this condition](https://github.com/qmaruf/scikit-learn/blob/2e0e022ed3a709f9cef37aa33e8ddc1677912aed/sklearn/feature_selection/from_model.py#L232).